core/domain/exp_services.py

```diff
@@ -665,6 +665,7 @@ def save_new_exploration(committer_id, exploration):
     }])
     user_services.add_created_exploration_id(committer_id, exploration.id)
     user_services.add_edited_exploration_id(committer_id, exploration.id)
+    user_services.record_user_created_an_exploration(committer_id)
```

Comment:
Ooh, yes. I did miss this, sorry. And the two lines above also confirm the interpretation we've gone with.

Would you mind fixing it? Thanks!